README.md

```diff
@@ -1,70 +1,63 @@
-# Getting Started with Create React App
+# Country-info
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A website . It is a Single Page App (SPA) that allows users to: get info about a country. get a weather for a specific cuntry.
+inspired by [this](https://www.behance.net/sakwadesignstudio) design
 
```

Comment:
- [ ] Right here in the README file, please attach a screenshot or two of your great app 😉 This would give the repo a more professional and good-looking shape. 🙏

Reply:
Well Done with fixes !!!